src/components/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,182 @@
+import { IconIcons, IconSize } from "@foundation/Icon";
+import { BadgeStyle } from "@components/Badge";
+
+const data = [
+    {
+        id: "tab-1",
+        label: "A tab",
+        content: (
+            <Text>
+                Bacon ipsum dolor amet spare ribs swine meatloaf, pastrami cupim tail leberkas frankfurter jowl chislic
+                shoulder. Frankfurter drumstick t-bone bacon ground round. Cupim pork loin shank kielbasa. Short loin
+                shank meatloaf tongue. Chicken sirloin swine ball tip. Turkey pork kevin burgdoggen meatball t-bone.
+            </Text>
+        ),
+    },
+    {
+        id: "tab-2",
+        label: "Disabled",
+        disabled: true,
+        content: "Rib-eye",
+    },
+    {
+        id: "tab-3",
+        label: "A long tab name",
+        content: (
+            <Text>
+                Sirloin prosciutto cow short loin, andouille bresaola doner fatback pork loin. Tongue ribeye kielbasa
+                pastrami. Meatball tail hamburger beef ribs beef ball tip pork chop buffalo short loin boudin chicken
+                pastrami ribeye picanha flank. Shoulder chicken pancetta shankle biltong rump hamburger kielbasa tri-tip
+                ham hock frankfurter.
+            </Text>
+        ),
+    },
+    {
+        id: "tab-4",
+        label: "An even longer tab name",
+        content: (
+            <div>
+                <Text>
+                    Porchetta burgdoggen kielbasa tenderloin boudin, ham cow pork loin salami. Pork chop swine meatloaf,
+                    sausage ground round tenderloin beef pork chislic chicken short loin landjaeger. Pig pork chop
+                    meatball picanha short ribs. Meatloaf buffalo shoulder, tail beef ribs corned beef short ribs ground
+                    round andouille pork belly landjaeger swine hamburger meatball. Ham hock turducken tri-tip picanha,
+                    buffalo leberkas pork loin chuck ground round.
+                    <Divider />
+                    <Button>Some button</Button>
+                </Text>
+            </div>
+        ),
+    },
+    {
+        id: "tab-5",
+        label: "Disabled",
+        disabled: true,
+        content: <Button>Sirloin</Button>,
+    },
+    {
+        id: "tab-6",
+        label: "Last tab",
+        content: <Text>Meat.</Text>,
+    },
+];
+
+// eslint-disable-next-line import/no-default-export
+export default {
+    title: "Components/Tabs",
+    component: Tabs,
+    argTypes: {
+        paddingX: {
+            options: Object.values(TabsPaddingX),
+            control: { type: "select" },
+        },
+        size: {
+            options: Object.values(TabSize),
+            control: { type: "select" },
+        },
+    },
+    args: {
+        paddingX: TabsPaddingX.Small,
+        size: TabSize.Small,
+    },
+} as Meta<TabsProps>;
+
+const TabTemplate: Story<TabsProps> = (args) => {
+    const [activeItemId, setActiveItemId] = useState("tab-1");
```

Comment:
```suggestion
    const [activeItemId, setActiveItemId] = useState(data[0].id);
```
I would do this in all other case where you have `tab-1`